### PR TITLE
PLT-4853 adding purge cache button to the UI.

### PR DESCRIPTION
**Pull request by coreyhulen**
#### Summary
Adding purge cache button to the UI.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4853

### CODE REVIEW

**Comment by coreyhulen** [comment]
Some UI to review for PMs.

**Comment by coreyhulen** [comment]
Taking to long for System Console UI review.  Moving to dev review since it's needed for testing.

File: webapp/components/admin_console/purge_caches.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                        values={{
+                            error: this.state.fail
+                        }}
+                    />
+                </div>
+            );
+        }
+
+        const helpText = (
+            <FormattedHTMLMessage
```
Comment (enahum) [inline webapp/components/admin_console/purge_caches.jsx]: can you make this `<FormattedMessage>` you also ain't using html tags in the string, so no need to use this one here, also we should try and avoid using `FormattedHTMLMessage`

File: webapp/components/admin_console/purge_caches.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                defaultMessage='This will purge all the in-memory caches for things like sessions, accounts, channels, etc. Deployments using High Availability will attempt to purge all the servers in the cluster.  Purging the caches may adversly impact performance.'
+            />
+        );
+
+        let contents = null;
+        if (this.state.loading) {
+            contents = (
+                <span>
+                    <span className='fa fa-refresh icon--rotate'/>
+                    {Utils.localizeMessage('admin.purge.loading', ' Loading...')}
```
Comment (enahum) [inline webapp/components/admin_console/purge_caches.jsx]: can you move this one to <FormattedMessage> too?

File: webapp/components/admin_console/purge_caches.jsx (hunk 1 above)
Comment (coreyhulen) [inline webapp/components/admin_console/purge_caches.jsx]: hah...will do (I copied the page from another control).

**Comment by coreyhulen** [comment]
ok @enahum ready for re-review.

**Pull request merged**